content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -45,67 +97,55 @@ team. These credentials are only used to generate the
 [authorization](#authorization) URL DVC will later prompt to visit in order to
 connect to the Google Drive.
 
-> Note that Google Drive API has usage limits/quotas per credentials in a
-> _project_ (which you can review in the
+> Note that the Google Drive API has usage limits/quotas per _project_ client
+> (which you can review in the
 > [OAuth consent screen](https://console.developers.google.com/apis/credentials/consent)).
 > Please keep this in mind when sharing them, or you may
 > [exceed the limits](https://developers.google.com/drive/api/v2/handle-errors?hl=ro#resolve_a_403_error_usage_limit_exceeded).
 
-## URL format for Google Drive remotes
+### Access Google Drive remote with OAuth credentials
 
-A GDrive DVC remote URLs is needed for the `dvc remote add` command. It can be
-constructed with a _base_, and an **optional** _path_ to a folder i.e.
-`gdrive://<base>/path/to/folder`. The base can be one of:
-
-1. `root` - indicates your topmost Google Drive directory.
-
-```dvc
-$ dvc remote add -d mygdroot gdrive://root
-$ dvc remote add mygdfolder gdrive://root/path/to/folder
-```
-
-2. Google Drive _Folder ID_
-
-To obtain the unique _Folder ID_ in question, navigate into that folder in your
-web browser, and find it in the address bar. For example, for
-`https://drive.google.com/drive/folders/0AIac4JZqHhKmUk9PDA`:
+Use the `dvc remote modify` command to set the credentials for each `gdrive://`
+remote, for example:
 
 ```dvc
-$ dvc remote add mygdfolder gdrive://0AIac4JZqHhKmUk9PDA
-$ dvc remote add mygdsubfolder \
-                 gdrive://0AIac4JZqHhKmUk9PDA/sub/path
+$ dvc remote add -d mygdfolder gdrive://root/path/to/folder
+$ dvc remote modify mygdfolder gdrive_client_id <client ID>
+$ dvc remote modify mygdfolder gdrive_client_secret <client secret>
 ```
 
-[Shared drives](https://developers.google.com/drive/api/v2/about-shareddrives)
-should be referenced by _Folder ID_ in the same way.
-
-3. `appDataFolder` - special
-   [application-specific data](https://developers.google.com/drive/api/v2/appdata)
-   folder only accessible by your application and hidden from the user.
+### Access Google Drive remote with a service account
```

Comment:
Simplify title - `Using service accounts`

Make it a separate section, put a link to it in the `Creating you app` and back to `Creating your app` as a prerequisite.

Reply:
OK, done.